openlibrary/utils/isbn.py

```diff
@@ -47,6 +47,10 @@ def isbn_10_to_isbn_13(isbn_10):
     isbn_13 = '978' + isbn_10[:-1]
     return isbn_13 + check_digit_13(isbn_13)
 
+def to_isbn_13(isbn):
```

Comment:
We should add some tests for this.
There probably should be some tests passing an isbn13 to `test_isbn_10_to_isbn_13()` and isbn10 to `test_isbn_13_to_isbn_10()`

`utils.normalise_isbn()` should be used more where hyphens are being stripped using custom code in other places.